mitodl/bootcamp-ecommerce
# chore!: Upgrade wagtail to 5.2.1(latest)

#### What are the relevant tickets?
Fixes https://github.com/mitodl/bootcamp-ecommerce/issues/1466

#### What's this PR do?
Upgrades the Wagtail Version to 5.0.1

#### How should this be manually tested?

- Checks should pass
- Check out the master branch and create some data like Bootcamps Homepage, and Subpages.
- Create some boot camps, BootCamp subpages, Certificates, and Signatories.
- Create some users and applications in different states.
-  Now stop containers and check out this branch 
- Build images and run `docker-compose run --rm web ./manage.py migrate`
- Now run the project and verify that there are no changes to the existing features/data.
- Now create some more data in CMS with the upgraded Wagtail version. Verify that the UI is updated.
- Now run `docker-compose down` to remove the db and the run the migrations `docker-compose run --rm web ./manage.py migrate`
- You can also run `docker-compose run --rm web ./manage.py configure_cms` and `docker-compose run --rm web ./manage.py seed_data`
- Now create some data on the fresh instance as mentioned above for the old instance.
- Verify that everything is fine.

#### Where should the reviewer start?

- Review PR
- Test for an existing instance
- Test for fresh instance

#### Any background context you want to provide?
Disabled Pylint due to issues with string-based model reference.

#### What GIF best describes this PR or how it makes you feel?
😴
